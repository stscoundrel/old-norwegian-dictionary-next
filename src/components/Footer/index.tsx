import styles from './Footer.module.scss'

export default function Footer() {
  return (
    <footer className={styles.section}>
      <div className="container">
        <p>Copyright Â© {new Date().getFullYear()}</p>
      </div>
    </footer>
  )
}
