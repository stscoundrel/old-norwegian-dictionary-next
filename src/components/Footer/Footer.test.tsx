import ReactDOM from 'react-dom/client'
import Footer from 'components/Footer'
import renderer from 'react-test-renderer'

const letters = [
  {
    slug: 'a',
    letter: 'A',
  },
  {
    slug: 'b',
    letter: 'B',
  },
]

describe('Footer component', () => {
  test('Does not crash', () => {
    const div = document.createElement('div')
    const root = ReactDOM.createRoot(div)
    root.render(<Footer letters={letters}/>)
  })

  test('Matches snapshot', () => {
    const tree = renderer.create(<Footer letters={letters} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
