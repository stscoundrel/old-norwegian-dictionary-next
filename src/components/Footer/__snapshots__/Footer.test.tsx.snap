// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Matches snapshot 1`] = `
<footer
  className="section"
>
  <div
    className="container"
  >
    <p>
      Copyright Â© 
      2021
    </p>
  </div>
</footer>
`;
